v2/support/secure_settings_sample_app/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Bag End Catering Helper",
+  "author": {
+    "name": "Frodo Baggins",
+    "email": "cgoddard@zendesk.com",
+    "url": "https://hobbiton.zendesk-staging.com"
+  },
+  "defaultLocale": "en",
+  "private": true,
+  "location": {
+    "support": {
+      "ticket_sidebar": "assets/iframe.html"
+    }
+  },
+  "parameters": [
```

Comment:
you need to define `domainWhitelist` in this manifest too. See [doc](https://developer.zendesk.com/apps/docs/developer-guide/using_sdk#using-secure-settings).